featurebyte/api/historical_feature_table.py

```diff
@@ -85,18 +85,6 @@ def feature_list(self) -> FeatureList:
         feature_list_id = self.cached_model.feature_list_id
         return FeatureList.get_by_id(feature_list_id)  # type: ignore
 
-    @property
-    def feature_names(self) -> list[str]:
-        """
-        List of feature names associated with the historical feature table.
-
-        Returns
-        -------
-        list[str]
-            List of feature names
-        """
-        return self.feature_list.feature_names
```

Comment:
Remove this as it has been declared in https://github.com/featurebyte/featurebyte/blob/main/featurebyte/models/historical_feature_table.py#L34-L46.